README.md

```diff
@@ -144,6 +134,38 @@ Deploy the OFT to Arbitrum Sepolia:
 pnpm hardhat lz:deploy --tags MyOFT --networks arbitrum-testnet
 ```
 
+### Verifying Deployed Contracts Manually
+
+The bytecode on-chain can be manually verified with the bytecode generated by solidity after running pnpm compile. For example, for the Mainnet OftAdapter:
+
+```
+# compile so we have the correct artifacts locally
+pnpm compile
+# get mainnet code for the deployed contract 
+cast code 0x5c7E91F4a8e58e701bFE621990bd6Fa4FD7c5379 --rpc-url $RPC_URL_1
+# get bytecode generated by solidity compilation
+cat artifacts/contracts/CowOftAdapter.sol/CowOftAdapter.json | jq .deployedBytecode -r
+```
+
+The output of the lsat 2 commands can then be pasted into an editor without word wrap for comparison.
```

Comment:
lsat has a typo

Also, I followed  the instructions and they are not easy to follow. 

How can I tell is correct? It ask me to check the bytes match, except some exceptions. Then I can't tell if this is good or not good. I do see small differences.

https://github.com/user-attachments/assets/2869dd99-a4ac-4f4d-96d5-43dac0ee07da